common/vm_wait_primary_nic_mac.yml

```diff
@@ -1,7 +1,12 @@
-# Copyright 2021-2023 VMware, Inc.
+# Copyright 2023 VMware, Inc.
 # SPDX-License-Identifier: BSD-2-Clause
 ---
-# Sometime we need to wait for a while to get mac address after create VM
+# Wait for VM's primary network adapter having MAC address
+#
+- name: "Initialize VM network adapters primary mac"
+  ansible.builtin.set_fact:
+    vm_primary_nic_mac: ""
+
 - name: "Check the mac address is available or not"
```

Comment:
Check the mac address is available or not -> Wait for VM primary network adapter MAC address

Reply:
Done. Thanks